lib/src/util/prop_conversion.dart

```diff
@@ -51,6 +51,16 @@ Map<K, V> unjsifyMapProp<K, V>(JsMap value) {
   return JsBackedMap.backedBy(value).cast();
 }
 
+/// Runs [jsifyMapProp] on every Dart Map in the [value].
+List<JsMap> jsifyMapListProp(List<Map> value) {
+  return value?.map(jsifyMapProp)?.toList();
+}
+
+/// Runs [unjsifyMapProp] on every JS Map in the [value].
+List<Map<K, V>> unjsifyMapListProp<K, V>(List<JsMap> value) {
+  return value?.map(unjsifyMapProp)?.toList()?.cast();
```

Comment:
I think the `cast` here might not be necessary
```suggestion
  return value?.map(unjsifyMapProp)?.toList();
```

Reply:
I'm getting this error without the `cast`:
```
A value of type 'List<Map<dynamic, dynamic>>' can't be returned from the function 'unjsifyMapListProp' because it has a return type of 'List<Map<K, V>>'.
```

Reply:
Oh wait I guess it works if I take out that `<K, V>` - is that what you were thinking?

Reply:
Oh I was thinking that the generics on `.map` would get inferred and passed to `unjsifyMapProp`, but it looks like they don't because of the `toList` call, so `dynamic` is getting used instead.

So I think the current implementation:
```dart
return value?.map((e) => unjsifyMapProp(e as JsMap))?.toList()?.cast();
```
is equivalent to:
```dart
return value?.map((e) => unjsifyMapProp<dynamic, dynamic>(e as JsMap))?.toList()?.cast<Map<K, V>>();
```

And what we really want is to pass the generics to `unjsifyMapProp` so the returned maps have the right type, which should also eliminate the need for `.cast()`:
```dart
return value?.map((e) => unjsifyMapProp<K, V>(e as JsMap))?.toList();
```

Reply:
Oh makes sense! Updated that!